### PR TITLE
Nov bugfixes

### PR DESCRIPTION
**Pull request by idanstark42**

### CODE REVIEW

File: client/src/components/scoresheet/menu/metadata_inputs.js [hunk 1]
```diff
@@ -11,9 +11,11 @@ export default {
     <div>
       <select ng-if="metadataInputs.data.current.teamNumber"
         ng-model="metadataInputs.data.current.matchId"
-        ng-class="{'disabled': metadataInputs.loadingMatches}"
+        ng-class="{'disabled': (metadataInputs.loadingMatches || metadataInputs.data.dontRequireMatch)}"
         ng-options="match._id as match.displayTextWithCompletion for match in metadataInputs.matches">
-        <option value="" disabled selected hidden>{{metadataInputs.loadingMatches ? 'Loading...' : 'Select Round'}}</option>
+        <option value="" disabled selected hidden>
+          {{metadataInputs.loadingMatches ? 'Loading...' : (metadataInputs.data.dontRequireMatch ? "Couldn't load matches, continue without them" : 'Select Round')}}
```
Comment (taltaub22) [inline client/src/components/scoresheet/menu/metadata_inputs.js]: This text gonna fit there?

File: client/src/controllers/scoring_controller.js [hunk 2]
```diff
@@ -36,10 +36,6 @@ class ScoringController {
     })
   }
 
-  bla () {
```
[the remark this answers is not in the document]
Comment (idanstark42) [reply client/src/controllers/scoring_controller.js]: Thanks

File: client/src/components/scoresheet/menu/metadata_inputs.js (hunk 1 above)
Comment (idanstark42) [reply client/src/components/scoresheet/menu/metadata_inputs.js]: Yes, this is the reason for the CSS change here

Comment (taltaub22) [reply client/src/components/scoresheet/menu/metadata_inputs.js]: Great

**Pull request merged**